### ISSUE TITLE
Unable to connect to docker error

### ISSUE THREAD

**Issue by saurabhdaware** [ameerthehacker/envase#1]
Hi, Ameer. Hope you're doing well!

I am getting this error when I run `yarn dev`

![A screen with unable to docker! written on it](https://user-images.githubusercontent.com/30949385/93572034-e766f280-f9b2-11ea-9638-20710326c1e3.png)

These are the logs I get in browser:
```
TypeError: Docker is not a constructor
./src/services/native/native.ts
src/services/native/native.ts:20

  17 |   allSettings: AllSettings;
  18 | } = window as any;
  19 | let { allSettings } = window as any;
> 20 | let dockerode: Dockerode = new Docker(allSettings);
  21 | 
  22 | ipcRenderer.on(IPC_CHANNELS.SAVE_SETTINGS, (evt, newSettings) => {
  23 |   allSettings = newSettings;
```

I followed steps from README and not sure if I missed something. 

I have `docker` command installed and accessible.

**Comment by ameerthehacker** [comment]
Hi @saurabhdaware, did you start the docker server?

**Comment by saurabhdaware** [comment]
Forgot to mention I am on Ubuntu. 

I have Docker engine running I am not sure if that is what you meant. 

![image](https://user-images.githubusercontent.com/30949385/93590428-1d64a080-f9cc-11ea-9dd1-383cc95b31cd.png)

**Comment by ameerthehacker** [comment]
@saurabhdaware yes I meant the same, let me know whenever you are free, let us get into a call to debug the issue as I have not tested this in linux

**Comment by saurabhdaware** [comment]
Yay! Got it working 🎉

So my docker needed root access apparently so I tried running electron with sudo as well and it worked 

So in one terminal, I executed
```sh
npx cross-env BROWSER=none react-scripts start
```
and in other 
```sh
sudo npx cross-env NODE_ENV=development electron ./main.js --no-sandbox
```
It said we have to pass `--no-sandbox` if we sudo electron.

and tada :D
![A screenshot with vscode running on localhost:8080](https://user-images.githubusercontent.com/30949385/93595657-c368d880-f9d5-11ea-8d29-3fde751e1f9f.png)

Thanks and great tool! I guess we can close this.
